starfish/src/main/java/org/maxandersen/Starfish.java

```diff
@@ -0,0 +1,238 @@
+package org.maxandersen;
+
+import io.quarkus.runtime.QuarkusApplication;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.io.File;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+public class Starfish implements QuarkusApplication {
+  @Override
+  public int run(String... args) throws Exception {   
+    
+    
+    for(String s:args)
+    System.out.println("\nTarget : "+s);//Printing Received Arguements for Debugging
+
+    if(args[0].equalsIgnoreCase("config")){
+    //Incase user wants to configure starfish
+    editConfig();//Calling function that enables to edit configuration
+    return 10; //Incase user typed "starfish config" we only want to edit configuration
+    }
+
+    //Calling function for Fetching installed Packages 
+    //fetch_installed_packages(); //This function will print all installed packages
+
+    //Configuration identifiers
+    String clone_path="";//Holds path to  destination Where the Repository Must Be Clonned
+    String ide="";  //Holds command for IDE to open upon
+    String userHome = System.getProperty( "user.home" ); //Get User Home Directory: /home/user_name
+
+    File configFile = new File(userHome + "/starfish-config.yml"); //Loading YAML
+
+
+  
+    
+    
+    //Reading Configuration File
+    try {
+        if (!configFile.exists()) {
+            //Incase no config file exists
+            System.out.println("\nThis is the first time you're using starfish...You'll need to  configure it");
+            editConfig(); //Calling function that lets user to configure 
+        }
+        
+        System.out.println("\nLoading configurations.....\n");
+        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+    
+        Config config = mapper.readValue(configFile, Config.class);
+        
+        ide=config.ide;
+        clone_path=config.clone_path;
+        if(ide==null||clone_path==null)editConfig(); //Incase of absence of configuration in file Launch Config 
+        
+        
+    }
+    catch (Exception e) {
+        e.printStackTrace();
+        System.out.print("Press any key to continue . . . ");
```

Comment:
This doesn't wait so this instructions seems wrong. Shouldnt it throw?

Reply:
This was working when I used it as a standalone program and made native executable directly with GraalVM.
when errors/exception occurred, it used to wait for user to enter any key.
Will resolve this or remove completely.

Reply:
removed